firmware/application/usb_serial_asyncmsg.hpp

```diff
@@ -47,10 +47,8 @@ class UsbSerialAsyncmsg {
 };
 
 /*Notes:
- * - Don't use MayhemHub since it currently not support real time serial output
- * - If you don't use this class linker will drop it so it won't use any space
- * - so delete all debug things before you push your code to production
- * - use this client to filter only PP devices: https://github.com/zxkmm/Pyserial-Demo-portapack
+ * - Don't use MayhemHub since it currently not support real time serial output (to hide some instructions from default features call (like screenframe) it filtered out by the response type)
```

Comment:
Maybe something like this sounds better:
"Don't use MayhemHub since it's currently not supporting real time serial output (it's hiding some instructions to support some features like screenshot, and it's filtering out answers by response type)

Reply:
Thank you, I will take these suggestions.